### PR TITLE
Fix log_struct usage

### PR DESCRIPTION
**Pull request by waprin**
Currently kwargs are not supported, though adding them would probably be a good feature.

### CODE REVIEW

File: docs/logging-usage.rst [hunk 1]
```diff
@@ -54,9 +54,9 @@ Write a dictionary entry to a logger.
    >>> from google.cloud import logging
    >>> client = logging.Client()
    >>> logger = client.logger('log_name')
-   >>> logger.log_struct(
-   ...     message="My second entry",
-   ...     weather="partly cloudy")  # API call
+   >>> logger.log_struct({
+   ...     'message':'My second entry',
+   ...     'weather':'partly cloudy'})  # API call
```
Comment (dhermes) [inline docs/logging-usage.rst]: Please use spaces between `:` and the value:

```
   >>> logger.log_struct({
   ...     'message': 'My second entry',
   ...     'weather': 'partly cloudy'})  # API call
```

**Comment by dhermes** [comment]
@waprin As an alternative to this, you could take all of the logging `snippets` and put them in `docs/logging_snippets.py` (though that's a bigger project)

**Comment by waprin** [comment]
Interesting, I'll take a look at that, seems like a great idea. For now we should prob fix what we have though.

**Comment by tseaver** [comment]
@waprin, @dhermes FWIW I have a `logging_snippets` branch locally where I have started that process.

**Comment by dhermes** [comment]
@tseaver Great news!

**Pull request merged**